app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -1,25 +1,38 @@
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui";
-import { Container, Header } from "neetoui/layouts";
+import { Delete } from "neetoicons";
+import { Button, PageLoader, Toastr } from "neetoui";
+import { Container, Header, SubHeader } from "neetoui/layouts";
+import { createContactEntity } from "utils";
 
 import EmptyState from "components/Common/EmptyState";
+import { CONTACTS } from "components/Dashboard/constants";
 
 import DeleteAlert from "./DeleteAlert";
 import Menu from "./Menu";
 import Create from "./Pane/Create";
 import Table from "./Table";
 
-import { CONTACTS } from "../constants";
-
 const Contacts = () => {
   const [loading, setLoading] = useState(true);
   const [showMenu, setShowMenu] = useState(false);
   const [showNewContactPane, setShowNewContactPane] = useState(false);
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [contacts, setContacts] = useState([]);
+  const [selectedContactIds, setSelectedContactIds] = useState([]);
+
+  const createContact = values => {
+    setContacts(contacts => [...contacts, createContactEntity(values)]);
+    Toastr.success("A new contact has been created successfully");
+  };
+  const removeContact = () => {
+    setContacts(contacts =>
+      contacts.filter(contact => !selectedContactIds.includes(contact.id))
+    );
+    Toastr.success("Deleted contact(s) successfully");
+  };
```

Comment:
Move to utils.

Reply:
I didn't move it intentionally, if I moved those to utils, then passing these functions to components would look ugly.😅
Should I move it anyway?

Reply:
Yes, you should move it. Pass the required arguments to these utils.